compute_tools/src/http/api.rs

```diff
@@ -142,8 +143,8 @@ async fn handle_configure_request(
     }
 
     let body_bytes = hyper::body::to_bytes(req.into_body()).await.unwrap();
-    let spec_raw = String::from_utf8(body_bytes.to_vec()).unwrap();
-    if let Ok(request) = serde_json::from_str::<ConfigurationRequest>(&spec_raw) {
+    let spec_raw = str::from_utf8(&body_bytes).unwrap();
+    if let Ok(request) = serde_json::from_str::<ConfigurationRequest>(spec_raw) {
```

Comment:
```suggestion
    if let Ok(request) = serde_json::from_slice::<ConfigurationRequest>(body_bytes) {
```

utf8 will be done regardless as json must always be utf8, and we have better error handling for the deserialization failing :)

Reply:
we use `unwrap` almost everywhere. I hope one day make it a bit fancier :D